doc/src/manual/interfaces.md

```diff
@@ -391,7 +391,7 @@ something other than 1), you should specialize `indices`. You should also specia
 so that the `dims` argument (ordinarily a `Dims` size-tuple) can accept `AbstractUnitRange` objects,
 perhaps range-types `Ind` of your own design. For more information, see [Arrays with custom indices](@ref).
 
-## [Strided Arrays]
+## [Strided Arrays](@id man-interface-strided-arrays)
```

Comment:
Is this tag used somewhere or what is the reason for this?

Reply:
No, I'm just following the convention in this file — and more importantly getting rid of the literal `[]` in the header.

Reply:
> ... getting rid of the literal [] in the header

Ah, right! So the alternative fix would just be to remove those I guess :)

Reply:
Or actually link to it. :)